SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1042,8 +1042,8 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 				const Worker_EntityId EntityId = PackageMap->GetEntityIdFromObject(ThisActor);
```

Comment:
Thinking through this, this could also happen if you've just referred to this actor before having created an entity for it. So I think we can cleanup this code and fix that edge case.

I think it should be;
1. If this is a startup actor we are authoritative over, and we haven't created an entity for it yet (check the view?), then create a tombstone actor. Dormant entities can remain just a list for those entities that *exist* and are dormant. Also worth noting that `CreateTombstoneEntity` will allocate a second entity id currently, which we don't want it to do. We should also consider how this will interact with skeleton entities cc: @improbable-dmitrii 
2. If the actor is dormant, and we're auth, we retire it.